gamestonk_terminal/cryptocurrency/overview/withdrawalfees_model.py

```diff
@@ -0,0 +1,150 @@
+"""Withdrawal Fees model"""
+from typing import Any, List
+import math
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+
+from gamestonk_terminal.helper_funcs import get_user_agent
+
+
+def get_overall_withdrawal_fees(top: int = 100) -> pd.DataFrame:
+    """Scrapes top coins withdrawal fees
+    [Source: https://withdrawalfees.com/]
+
+    Parameters
+    ----------
+    top: int
```

Comment:
This threshold arg is called "limit" in sia. We might want to check how it's named in other menus and make it consistent